app/view/showCompany/showCompany.js

```diff
@@ -30,4 +31,59 @@ angular.module('csrLookupApp.showCompany', ['ngRoute'])
   }, function errorCallback(response) {
     $scope.evidence_records = null
   });
+  
+   function fillWikipediaBox(companyWikipediaName)
+   {
+       var wikiData;
+        $.ajax({
+            type: "GET",
+            url: "http://en.wikipedia.org/w/api.php?action=parse&format=json&prop=text&section=0&callback=?&page="+companyWikipediaName,
+            contentType: "application/json; charset=utf-8",
+            async: false,
+            dataType: "json",
+            success: function (data, textStatus, jqXHR) {
+                console.log(data);
+                wikiData = data;
+                $("#wikipediaBox").html('');
+                if(wikiData.parse)
+                {
+                    var html = wikiData.parse.text['*'];
+                    var parsedHtml = $.parseHTML( html );
+                    var infobox = $(parsedHtml[0]).find('table.infobox.vcard');
+                    //infobox[0].attributes['style'] = ''
+                    //removing the width style from the result wikipedia infobox;
```

Comment:
Styling this is a pain, huh? How does wikipedia do it? We can make the column wider if it helps...

Reply:
yeah it's pain, I moved the style to our style sheet but gave it !important to override the wikipedia inline style